CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/SaveGraphPlugin.java

```diff
@@ -45,20 +46,26 @@
 @NbBundle.Messages("SaveGraphPlugin=Save Graph")
 public class SaveGraphPlugin extends SimplePlugin {
 
+    private static final Logger LOGGER = Logger.getLogger(SaveGraphPlugin.class.getName());
+
     public static final String GRAPH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "graphId");
-    public static final String FILE_PATH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, System.getProperty("user.home"));
+    public static final String FILE_PATH_PARAMETER = PluginParameter.buildId(SaveGraphPlugin.class, "saveFilePathId");
 
     @Override
     public PluginParameters createParameters() {
         final PluginParameters parameters = new PluginParameters();
 
         final PluginParameter<StringParameterValue> graphIdParameter = StringParameterType.build(GRAPH_PARAMETER);
-        final PluginParameter<StringParameterValue> filePathParameter = StringParameterType.build(FILE_PATH_PARAMETER);
+        final PluginParameter<StringParameterValue> saveFilePath = StringParameterType.build(FILE_PATH_PARAMETER);
+
         graphIdParameter.setName("graphId");
         graphIdParameter.setDescription("The Id of the graph");
         parameters.addParameter(graphIdParameter);
-        filePathParameter.setName("SaveGraphPluginFilePath");
-        filePathParameter.setDescription("Save Graph Plugin File Path");
+
+        saveFilePath.setName("SaveGraphPluginFilePath");
```

Comment:
I think I must have confused you so my apologies :). I was ok with the variable name being filePathParameter (makes sense and matches the string you were building with). I was thinking more changing the name set here from "SaveGraphPluginFilePath" to "saveFilePath". It keeps the plugin part out of the name which would be consistent with most plugin parameters.

Reply:
Oh right that makes more sense 😆 . I've literally just woken up so that's on me for misunderstanding.